examples/eval/ackermann.sf

```diff
@@ -12,4 +11,4 @@ val ackermann = fix(ackermann =>
   }
 );
 
-assert(ackermann 2 2 == 7)
```

Comment:
Why remove the assertion?

Reply:
It was for testing by looking at each step of the parteval, so the assert added more steps, I should have rectified that, will do